src/mappings/clearingHouse.ts

```diff
@@ -136,6 +137,11 @@ export function handlePositionChanged(event: PositionChangedEvent): void {
     protocol.blockNumber = event.block.number
     protocol.timestamp = event.block.timestamp
 
+    // update ProtocolDayData
+    const protocolDayData = getOrCreateProtocolDayData(event)
+    protocolDayData.tradingFee = protocolDayData.tradingFee.plus(positionChanged.fee)
+    protocolDayData.tradingVolume = protocolDayData.tradingVolume.plus(abs(positionChanged.exchangedPositionNotional))
```

Comment:
Should we also do this in `handlePositionClosed`?

Reply:
updated